test/functional/wallet_exported_watchonly.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env python3
+# Copyright (c) 2025-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+import os
+
+from test_framework.descriptors import descsum_create
+from test_framework.key import H_POINT
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    assert_not_equal,
+    assert_raises_rpc_error,
+)
+from test_framework.wallet_util import generate_keypair
+
+KEYPOOL_SIZE = 10
+
+class WalletExportedWatchOnly(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+        self.extra_args = [[], [f"-keypool={KEYPOOL_SIZE}"]]
+
+    def setup_network(self):
+        # Setup the nodes but don't connect them to each other
+        self.setup_nodes()
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def export_and_restore(self, wallet, export_name):
+        export_path = os.path.join(self.export_path, f"{export_name}.dat")
+        res = wallet.exportwatchonlywallet(export_path)
+        assert_equal(res["exported_file"], export_path)
+        self.online.restorewallet(export_name, res["exported_file"])
```

Comment:
The assertion below (same as found in `wallet_migration.py`) fails.
Is this expected ?

```suggestion
        # Assert no rescan occurs when restoring the exported watch-only wallet.
        with self.online.assert_debug_log(unexpected_msgs=["Rescanning"]):
            self.online.restorewallet(export_name, res["exported_file"])
```

Reply:
Yes. The nodes in this test are not always in sync with each other.